docs-kits/kits/Data Chain Kit/page_adoption-view.md

```diff
@@ -1,39 +1,137 @@
+| Portal                             | The Catena-X Portal and Marketplace is the heart of Catena-X and the entry point for all activities in the automotive network/value chain. Every user of the automotive value chain, no matter if it is a consumer, app provider, IT administrator or IT support will connect via the portal to the value bringing services. The whole eco-system on the automotive value chain is connected at one place.                                                                                                                                                                      |                                                                                                                                                                                                        |
+| EDC Discovery Finder               | The EDC dataspace discovery interface is a CX network public available endpoint which can get used to retrieve edc endpoints and the related BPNs, as well as search for endpoints via the BPN                                                                                                                                                                                                                                                                                                                                                                                |                                         
+
+## Business Process
+
+The DataChainKit acts as a middleware between consumers and manufacturers. This section describes the Business Process of DataChainKit. Who are its users, and with which other systems does it interact with.
+
+### Consumer
+
+The DataChainKit by IRS API is the essential component of the business app Trace-X FOSS to be able to build data chains in upwards and downwards direction for BOM lifecycles asBuilt and asPlanned. 
+The DataChainKit by IRS API is being consumed by the dismantler dashboard and other parties which are part of the Catena-X network. They need to provide valid credentials issued by the Catena-X Portal/IdP. Additionally, they must provide a base global asset identifier to retrieve information for as well as configuration details for the view on that information.
+
+### Catena-X network
+
+The IRS retrieves data from the Catena-X network (using the necessary infrastructure, see Technical Context), aggregates it and provides it back to the consumers. This connection is mandatory. If the Catena-X services are unavailable, the IRS cannot perform any work.
+
+As of now, the IRS uses it technical user credentials provided by the portal to gather the required data. This might be changed to use the consumer credentials in the future.
```

Comment:
```suggestion
As of now, the IRS uses technical user credentials provided by the portal to gather the required data. This might be changed to use the consumer credentials in the future.
```